lib/api_base_rest/cache/cache_manager.dart

```diff
@@ -0,0 +1,22 @@
+import "package:flutter_cache_manager/flutter_cache_manager.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:riverpod_annotation/riverpod_annotation.dart";
+
+part "cache_manager.g.dart";
+
+@Riverpod(keepAlive: true)
+CacheManager cacheManagerSingleton(Ref ref) {
+  return CacheManager(
+    CacheManagerConfig.jsonCacheManagerConfig,
+  );
+}
+
+abstract class CacheManagerConfig {
+  static final jsonCacheManagerConfig = Config(
+    "JsonCacheManager7Days",
+    stalePeriod: const Duration(days: 7),
```

Comment:
so I'm thinking how can we use different TTL durations for different endpoints

Reply:
So I'm thinking either separate cache managers for every endpoint or separate cache managers for different TTL durations

Reply:
e.g. maybe smth like this: 
```dart
@Riverpod(keepAlive: true)
CacheManager restCacheManage(Ref ref, int ttlDays) {
  return CacheManager(
    Config(
      "JsonCacheManager${ttlDays}Days",
      stalePeriod: Duration(days: ttlDays),
      maxNrOfCacheObjects: 10,
    ),
  );
}
```